cdhweb/projects/templates/projects/project_list.html

```diff
@@ -1,6 +1,6 @@
 {% extends "base.html" %}
 
-{% block page-subtitle %}{{ title }} | {% endblock %}
+{% block page-title %}{{ title|default:'People' }}{% endblock %}
```

Comment:
Do we need a default? And if so, this seems like the wrong default

Reply:
good catch!! the other list templates had defaults, so I put one here, but obvi didn't edit after pasting